### DISCUSSION TITLE
Branch Rule

### DISCUSSION THREAD

**Discussion by doho-ho** [SENAGI/SENAGI#3] [Ideas]
Gitlab에서 Github로 옮겨온 만큼 Github의 branch rule기능을 활용해보아요!

main : 운영 버전
release : 배포 버전
dev : 개발 버전
feat : dev에서 파생된 브랜치

Merge는 [ feat > dev > release > main ] 순으로 갈 것 같습니다.

우선 생각하는 것
1. feat > dev (PR approve 1명 이상)
2. dev > release (배포 담당자 && 2명 이상)
3. release > main (제한된 인원만 가능하도록 허용) - maintainer

인데 의견 달아주세요~
@alskal1 @haiyinde @ethanKim93

**Comment by alskal1** [comment]
좋습니다!

**Comment by haiyinde** [comment]
좋아요!
